src/postgres/src/test/regress/yb_pg_join_schedule

```diff
@@ -0,0 +1,20 @@
+# src/test/regress/yb_pg_join_schedule
+#
+####################################################################################################
+# This includes tests for Join Clause
+####################################################################################################
+# Dependencies for `yb_pg_join`
+test: yb_pg_int4
+test: yb_int2
+test: yb_pg_int8
+test: yb_float8
+test: yb_text
+
+
+# Chain of dependencies for `yb_pg_join`
+test: yb_pg_create_function_1
+test: yb_pg_create_type
+test: yb_pg_create_table
+test: yb_pg_copy
```

Comment:
Again, can you verify this dependency is valid?  Ditto for the rest of the schedules.

Reply:
It is a chain of dependencies needed to fill the tables for the tests.